framework/components/ecl-breadcrumbs/_breadcrumbs.scss

```diff
@@ -1,268 +1,55 @@
-/* stylelint-disable */
+/*
+ * Breadcrumbs
+ * @define breadcrumbs
+ */
+
 $breadcrumb-heigth: 2.778em;
 
-.breadcrumb {
-  position: relative;
-  margin: 0;
-  padding: 0;
-  background: none;
-  font-size: 0.833em;
-  font-weight: 600;
+.ecl-breadcrumbs {
+  @include ecl-hidden-print();
+
   border-bottom: 1px solid #fff;
-  border-radius: 0;
+  font-size: map-get($ecl-font-size, 's');
   line-height: normal;
+  margin: 0;
```

Comment:
Remove the `margin` and `padding` here doesn't seem to break anything. Is it true?